### ISSUE TITLE
Handle navigator.mediaSession

### ISSUE THREAD

**Issue by lethak** [lethak/digitally_imported_userscript#40]
Ideas:
* Add previous / skip track controls
* Custom background...

Read more:
* https://developer.mozilla.org/en-US/docs/Web/API/MediaSession/metadata
* https://techcrunch.com/2020/01/16/chrome-gets-global-media-controls/
* https://stackoverflow.com/questions/59828493/how-to-customize-global-media-controls-in-google-chrome-question-and-answer

Will not be compatible with IE, Safari and exotic browsers.

Proof of concept code

```javascript
if ('mediaSession' in navigator) {
    navigator.mediaSession.metadata = new MediaMetadata({
        title: "TITLE",
        artist: "ARTIST",
        album: "ALBUM",
        artwork: [{
            sizes: "320x180",// <- MUST BE EXACTLY!
            src: "https://i.ytimg.com/vi/yAruCvT7P7Y/hqdefault.jpg?sqp=-oaymwEZCNACELwBSFXyq4qpAwsIARUAAIhCGAFwAQ==&rs=AOn4CLAfHWw5BHrQugGsdPYy4eIXcqMTnQ",
            type: ""
        }]
    });

    navigator.mediaSession.setActionHandler('play', function () { });
    navigator.mediaSession.setActionHandler('pause', function () { });
    navigator.mediaSession.setActionHandler('seekbackward', function () { });
    navigator.mediaSession.setActionHandler('seekforward', function () { });
    navigator.mediaSession.setActionHandler('previoustrack', function () { });
    navigator.mediaSession.setActionHandler('nexttrack', function () { });
}```

**Comment by lethak** [comment]
Will be included in the next release

**Comment by lethak** [comment]
This will be included in the next release

![Support for browser's Global Media Controls](https://i.imgur.com/SG3lYkP.png)

**Comment by lethak** [comment]
Implemented in [v2020.10.642](https://github.com/lethak/LTKDIFMU-userscript/tree/v2020.10.642)
